examples/02_payment.rb

```diff
@@ -1,15 +1,41 @@
 require 'stellar-sdk'
 
-account = Stellar::Account.from_seed("SBXH4SEH32PENMMB66P4TY6LXUIFMRVFUMX2LJC3P2STHICBJLNQJOH5") 
 client  = Stellar::Client.default_testnet()
 
-# create a random recipients
-recipient = Stellar::Account.random
+puts "Creating random sender..."
+from = Stellar::KeyPair.random
+client.friendbot(from)
+puts "Creating random recipient..."
+recipient = Stellar::KeyPair.random
+client.friendbot(recipient)
 
-# make a payment
-client.send_payment({
-  from:   account,
-  to:     recipient,
-  amount: Stellar::Amount.new(100_000_000)
-}) # => #<OK>
+puts "Retrieving account's current sequence number..."
+seq_num = client.account_info(from.address).sequence.to_i
 
+puts "Constructing transaction..."
+# construct TransactionBuilder and payment Operation
+builder = Stellar::TransactionBuilder.new(
+  source_account: from,
+  sequence_number: seq_num + 1
+)
+# Note: if you want to send a non-native asset, :amount must take the form: 
+# [<:alphanum12 or :alphanum4>, <code>, <issuer keypair>, <amount>]
+payment_op = Stellar::Operation.payment({
+  destination: recipient,
+  amount: [:native, 100]
+})
+
+# add payment to transaction and set a 600ms timeout
+tx = builder.add_operation(payment_op).set_timeout(600).build()
+# sign transaction and get xdr
+envelope = tx.to_envelope(from).to_xdr(:base64)
```

Comment:
It's a bit weird we need to pass the `from` in here again. Is the reason we pass it here for signing? It'd be great to make that explicit I think. But that's not a symptom of this PR.

Reply:
I agree, it confused me too when I started using the SDK. `to_envelope`'s arguments are signers on transaction envelope.